samza-azure/src/main/java/org/apache/samza/coordinator/AzureJobCoordinator.java

```diff
@@ -90,6 +93,7 @@
   private RenewLeaseScheduler renewLease;
   private LeaderBarrierCompleteScheduler leaderBarrierScheduler;
   private StreamMetadataCache streamMetadataCache = null;
+  private SystemAdmins systemAdmins = null;
```

Comment:
Where is this used?

Reply:
I think it is used in start() and stop().

Reply:
Fixed as suggested.